src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/listHeader.tsx

```diff
@@ -2,23 +2,45 @@ import React from 'react';
 import styled from '@emotion/styled';
 
 import {t} from 'app/locale';
+import Tooltip from 'app/components/tooltip';
 import space from 'app/styles/space';
+import {IconSwitch} from 'app/icons';
 
-import {Grid, GridCell} from './styles';
+import {GridCell} from './styles';
 
-const ListHeader = () => (
-  <StyledGrid>
+const getTimeTooltipTitle = (displayRelativeTime: boolean) => {
+  if (displayRelativeTime) {
+    return t('Switch to ISO 8601');
+  }
+  return t('Switch to relative');
+};
+
+type Props = {
+  onSwitchTimeFormat: () => void;
+  displayRelativeTime: boolean;
+};
+
+const ListHeader = React.memo(({onSwitchTimeFormat, displayRelativeTime}: Props) => (
+  <React.Fragment>
     <StyledGridCell>{t('Type')}</StyledGridCell>
-    <StyledGridCellCategory>{t('Category')}</StyledGridCellCategory>
+    <Category>{t('Category')}</Category>
     <StyledGridCell>{t('Description')}</StyledGridCell>
     <StyledGridCell>{t('Level')}</StyledGridCell>
-    <StyledGridCell>{t('Time')}</StyledGridCell>
-  </StyledGrid>
-);
+    <Time onClick={onSwitchTimeFormat}>
+      <Tooltip title={getTimeTooltipTitle(displayRelativeTime)}>
+        <StyledIconSwitch size="xs" />
+      </Tooltip>
+      <span> {t('Time')}</span>
+    </Time>
+  </React.Fragment>
+));
 
 export default ListHeader;
 
 const StyledGridCell = styled(GridCell)`
+  position: sticky;
+  z-index: 2;
```

Comment:
Is there a zindex in theme we can use?

Reply:
at the moment our zIndex in theme are:

```
  // Try to keep these ordered plz
  zIndex: {
    traceView: {
      spanTreeToggler: 900,
      rowInfoMessage: 900,
      dividerLine: 909,
      minimapContainer: 999,
    },

    header: 1000,
    errorMessage: 1000,
    dropdown: 1001,

    dropdownAutocomplete: {
      // needs to be below actor but above other page elements (e.g. pagination)
      // (e.g. Issue Details "seen" dots on chart is 2)
      // stream header is 1000
      menu: 1007,

      // needs to be above menu
      actor: 1008,
    },

    globalSelectionHeader: 1009,
    sidebarPanel: 1019,
    sidebar: 1020,
    orgAndUserMenu: 1030,

    // Sentry user feedback modal
    sentryErrorEmbed: 1090,

    // If you change modal also update shared-components.less
    // as the z-index for bootstrap modals lives there.
    modal: 10000,
    toast: 10001,

    // tooltips and hovercards can be inside modals sometimes.
    tooltip: 10002,
  },
```

Honestly, I find my self very often confused about them.. in this case I am not sure which one to use... dropdown? errorMessage? ... neither of them is exactly what this code is about

Reply:
should I create a new property for breadcrumbs? We should definitely review the Index as well

Reply:
@priscilawebdev I think creating a new one is fine -- the goal of the z-indexes in theme is to be to have an index of all z-indexes that are in use. Ideally, this lets people see the existing z-indexes so that they can use the proper index without the arms race of increasing z-index.

Reply:
ok. I've created the following object:

```
breadcrumbs: {
      header: 2,
      gridCellError: 1,
      iconWrapper: 1,
    },
```